app/handlers/profile.js

```diff
@@ -1,20 +1,19 @@
-'use strict';
+"use strict";
```

Comment:
Again all single quotes are being replaced. this will become a giant pain in the butt in the future.

Reply:
Will keep in mind. My bad.